### ISSUE TITLE
Populate `rpki_vrps` metric

### ISSUE THREAD

**Issue by racompton** [ties/rpki-client-web#79]
It looks like `rpki_vrps` is defined but there are no values returned.

**Comment by ties** [comment]
For my instances the `rpki_vrps` metric is defined. One example is http://130.89.168.83:8890/metrics .

If you can share the configuration file + json output I will take a further look.

**Comment by racompton** [comment]
I only see `rpki_vrps{ta="arin"} 63428.0` in that endpoint that you shared.  I was hoping for a total number of VRPs.  
Here is my json file: https://drive.google.com/file/d/19jSqjksB_vJD1xjUpLR2Io3JR2KW2bCz/view?usp=sharing

Here are the non-commented lines from my config.yml file:
```
cache_dir: /opt/rpki-client-web/cache
output_dir: /opt/rpki-client-web/output
interval: 1200
timeout: 1175
port: 8323
rpki_client: /usr/sbin/rpki-client
additional_opts: []
trust_anchor_locators:
  - /etc/tals/afrinic.tal
  - /etc/tals/apnic.tal
  - /etc/tals/lacnic.tal
  - /etc/tals/ripe.tal
  - /etc/tals/arin.tal
 ```

**Comment by ties** [comment]
> My json file produced by rpki-client is in `/var/lib/rpki-client/`. Do I need to tell rpki-client-web where it is in that config.yml file?

Yes. The output file produced by rpki-client-web should be the one that it uses for the data.  It expect this to be in the `output_dir` from settings.

But the output file in `/var/lib/rpki-client` puzzles me. Could it be that this is produced by, for example, cron?

**Comment by racompton** [comment]
Yes, I'm using a crontab entry to have `rpki-client` run.  The output file`/var/lib/rpki-client/json` is what I have configured it to generate.
My cron.hourly entry looks like this: 
`/usr/sbin/rpki-client -j /var/lib/rpki-client/ -d /var/lib/rpki-client/cache/`

I tried setting:
```
cache_dir: /var/lib/rpki-client/cache/
output_dir: /var/lib/rpki-client/ 
```
but this didn't seem to fix the issue.  
Any idea what I need to do to get these additional metrics?
